test/python/expected/pyodbc/ddl_test.out

```diff
@@ -0,0 +1,166 @@
+SET ANSI_NULLS ON
```

Comment:
1. Rename the test file based on the object type which is "Table" here. 
2. Add tests for all the remaining objects.

Reply:
Can you add comment with tests to show what the test is actually meant for?